docs/writing/posts/learning.md

```diff
@@ -0,0 +1,153 @@
+
+### You only need to be a little bit ahead
+
+I found myself in a situation where hundreds of people messaged me with their machine learning and data science questions, not knowing I literally no better than them. However, I would diligently search Google and YouTube to find answers and call them to discuss the topics. Just like in highschool, although I was only slightly more knowledgeable than them, having this audience allowed me collect interesting questions and find answers.
+
+By the time I started the Data Science Club, my goal was simply to teach as much as possible. I would give a couple of lengthy lectures every semester and organize career panels. I practiced reviewing résumés to develop a better understanding of how to write my own. Additionally, I honed my interview skills by conducting mock interviews.
+
+Eventually, I started dedicating half my Saturday and would just study while on Google Meet sessions and if anyone popped in I'd learn more about their experiences, answer their questions whenever possible, and learn from their insights. Many of those people are now my friends I've known for over 10 years.
+
+### Keep Practicing
+
+Every opportunity to teach, learn, and practice is a chance to improve. Being able to explain something well results in people asking out-of-the-box questions that you wouldn't have thought of. I find that the goal of teaching is secretly for the audience to bring all the idiosyncratic edge cases of your knowledge to your attention, so you can learn from them. It's like a knowledge pyramid scheme.
+
+## Leverage
+
+My whole strategy for learning can be summarized into a set of tactics. I worked very hard in the beginning to get a bit ahead, built a peer group that is a few steps behind me, and then taught them. Rinse and repeat. Their role in this ecosystem is to bring me the edge cases I wouldn't have thought of, and my job is to learn alongside them, always a few steps ahead, and then teach them.
+
+Even if I'm only two weeks ahead, helping them out means spending a few hours helping them get two weeks ahead. If you're two weeks ahead and spend one hour teaching 30 people, you just provided 60 human weeks of value and years of value in one hour. That's a pretty good exchange rate.
+
+## The real world
+
+That basically summarizes how I thought about learning in a school setting where because roughly speaking, everyone is somewhere on the assembly line, the system I used to make progress in new hobbies was very differen since it's rare to enter a new place as an adult and find you have access to a large group of people who are all trying to learn the same thing with the same level of intensity as a group of high achieving students, many of those same people will not pursuit any hobbies as they decide to focus on their careers.
+
+The following section will mostly be some woo woo so I'm going to premptively apologize to the hacker news crowd.
```

Comment:
Hacker News? Unless you're intentionally making it lower case to address like hacker related news and not news.ycombinator.com

Also I don't think this section is woo!